src/CognitiveServices/CognitiveServices/CognitiveServicesAccount/CognitiveServicesAccountBaseCmdlet.cs

```diff
@@ -84,6 +84,11 @@ protected void RunCmdLet(Action action)
             }
             catch (ErrorException ex)
             {
+                if (ex.Body == null)
```

Comment:
When service response empty string, Body is null. so display ex.Message instead.

Other changes are all just SDK update.